### DISCUSSION TITLE
ارتباط ناپایدار از نسخه 0.17.12 به بعد

### DISCUSSION THREAD

**Discussion by seyda2020** [hiddify/hiddify-app#708] [General]
سه نسخه 0.18، 1.0 و 1.1 ارتباط پایداری نداشتن و زمان استفاده از گوگل پلی نمی تونستن برنامه ها رو دانلود کنن.
 بهترین نسخه ای که تا این لحظه برای اندروید استفاده کردم، 0.17.5 بوده و برای ویندوز 0.17.12

**Comment by Coderx7** [comment]
برای من بهترین نسخه 0.15.4 بوده رو پی سی و رو اندروید 0.16.16. بقیه داغون
